contracts/core/fee-calculators/PerformanceFeeCalculator.sol

```diff
@@ -208,6 +213,34 @@ contract PerformanceFeeCalculator is IFeeCalculator {
         return streamingFee.add(profitFee);
     }
 
+    /*
+     * Validate then set new streaming fee.
+     *
+     * @param  _newFeePercentage       Fee type and new streaming fee encoded in bytes
+     */
+    function adjustFee(
+        bytes calldata _newFeePercentage
+    )
+        external
+    {
+        (
+            FeeType feeIdentifier,
+            uint256 feePercentage
+        ) = parseNewFeeCallData(_newFeePercentage);
+
+        // Since only two fee options and anything feeType integer passed that is not 0 or 1 will revert can
+        // make this a simple if...else... statement
+        if (feeIdentifier == FeeType.StreamingFee) {
+            validateStreamingFeePercentage(feePercentage);
+
+            feeState[msg.sender].streamingFeePercentage = feePercentage;
+        } else {
```

Comment:
I feel better if we have explicit 
```
else if (feeIdentifier == 1)
```

Reply:
I would too but then we have coverage issues cause anything that doesn't fall under those two options would have reverted before and coverage will say there's a branch that hasn't been explored.

Reply:
We could add a else block that reverts "Invalid feeidentifier"

Reply:
That's what I'm saying it won't get to that point cause say you try to assign 3 to the enum it will revert when you're parsing the call data.

Reply:
Since there's no switch case in Solidity we either have to be ok with this pattern or skip coverage on this line